cdap-api/src/main/java/co/cask/cdap/api/dataset/table/Table.java

```diff
@@ -29,6 +29,19 @@
  */
 public interface Table extends BatchReadable<byte[], Row>, BatchWritable<byte[], Put>, Dataset {
   /**
+   * Property set to configure time-to-live on data within this dataset. The value given is in milliseconds.
+   * Once a cell's data has surpassed the given value in age,
+   * the cell's data will no longer be visible and may be garbage collected.
+   */
+  String PROPERTY_TTL = "dataset.table.ttl";
+
+  /**
+   * Property set to configure read-less increment support for a dataset.  When not set, calling the
+   * {@link Table#increment(byte[], byte[], long)} method will result in a normal read-modify-write operation.
+   */
+  String PROPERTY_READLESS_INCREMENT = "dataset.table.readless.increment";
+
```

Comment:
+1 on keeping the string value unchanged. That keeps existing tables properties valid.